packages/mermaid/src/diagrams/git/gitGraphRenderer.js

```diff
@@ -388,49 +396,61 @@ const findLane = (y1, y2, depth = 0) => {
  * Draw the lines between the commits. They were arrows initially.
  *
  * @param {any} svg
- * @param {any} commit1
- * @param {any} commit2
+ * @param {any} commitA
+ * @param {any} commitB
  * @param {any} allCommits
  */
-const drawArrow = (svg, commit1, commit2, allCommits) => {
-  const p1 = commitPos[commit1.id];
-  const p2 = commitPos[commit2.id];
-  const overlappingCommits = hasOverlappingCommits(commit1, commit2, allCommits);
-  // log.debug('drawArrow', p1, p2, overlappingCommits, commit1.id, commit2.id);
+const drawArrow = (svg, commitA, commitB, allCommits) => {
+  const p1 = commitPos[commitA.id]; // arrowStart
+  const p2 = commitPos[commitB.id]; // arrowEnd
+  const arrowNeedsRerouting = shouldRerouteArrow(commitA, commitB, p1, p2, allCommits);
```

Comment:
I feel like this function signature can be simplified down to
```
shouldRerouteArrow(commitA, commitB, allCommits);
```
I know that it may seem redundant to get p1 and p2 inside the function once again but the access time of commitPos is O(1) and there is no cases when we have to pass positions independently from commits.

What do you think?

Reply:
`shouldRerouteArrow` function violates DRY anyway, because it uses `dir` property. In case we consider this function as a method or instance-related function it can use all object attributes without referencing the object directly (this, self ...etc). In case we consider this function as a pure one, we have to pass everything as arguments including `dir`. Right now this is neither of these two. That is why I have suggested to stick to previous signature.

Btw, I am ok with how it looks now. Simply looking for better options.